config/locales/time/en.yml

```diff
@@ -2,6 +2,7 @@
 en:
   date:
     day_names:
+      -  # empty item to have correct weekday offset for %A
```

Comment:
There's a couple places in the code where `t('date.day_names')[0]` is hardcoded into templates; you should probably review everywhere that `date.day_names` is referenced and make sure that we're still displaying the day name we were before this change.

Reply:
good call out, i'll take a look at that!
---ack i wrote this yesterday but failed to click comment